packages/yoastseo/src/languageProcessing/researches/getLongCenterAlignedText.js

```diff
@@ -0,0 +1,84 @@
+import { map } from "lodash-es";
+import { languageProcessing } from "yoastseo";
+const { sanitizeString } = languageProcessing;
+
+const centerAlignClass = "class=\"has-text-align-center\"";
+const paragraphsRegex = /<p(?:[^>]+)?>(.*?)<\/p>/ig;
+const headingsRegex = /<h([1-6])(?:[^>]+)?>(.*?)<\/h\1>/ig;
+
+/**
+ * Matches the paragraphs in <p>-tags and returns the text in them.
```

Comment:
I guess the function description is not accurate? since this helper is used for both paragraph and subheadings right? @agnieszkaszuba

Reply:
yes, in my latest commit I fixed the JSDoc comments :D